src/test/test_utils.h

```diff
@@ -0,0 +1,60 @@
+#ifndef TEST_TEST_UTILS_H_
+#define TEST_TEST_UTILS_H_
+
+#include "database/cost_matrix_database.h"
+#include "database/online_database.h"
+#include "localization_protos.pb.h"
+
+#include "gtest/gtest.h"
+
+#include <filesystem>
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace test {
+
+inline image_sequence_localizer::Feature
+createFeatureProto(const std::vector<double> &values) {
+  image_sequence_localizer::Feature feature_proto;
+  feature_proto.set_type("debug");
+  feature_proto.set_size(values.size());
+  for (double value : values) {
+    feature_proto.add_values(value);
+  }
+  return feature_proto;
+}
+
+inline void
+createFeatureFile(const std::filesystem::path &dir, const std::string &name,
+                  const image_sequence_localizer::Feature &feature_proto) {
+
+  const std::filesystem::path filename = dir / name.c_str();
```

Comment:
nit: no need for the `c_str()` here if I'm not mistaken.

Reply:
That's correct. No need.